firmware/src/io/store.cpp

```
@@ -0,0 +1,203 @@
+  const char* key = line;
+  const char* text = separator + 1;
+  int value = 0;
+  int digits = 0;
+  for (const char* c = text; is_digit(*c); ++c) {
+    value = value * 10 + (*c - '0');
+    if (++digits > 3) return;
+  }
+  if (digits == 0 || text[digits] != '\0') return;
+  const bool flag = value <= 1;
+  if (std::strcmp(key, "song") == 0 && value >= kFirstSlot && value <= kLastSlot) settings.song = value;
+  if (std::strcmp(key, "brightness") == 0 && value <= 100) settings.brightness = value;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)